database/migrations/2016_11_18_200000_create_combined_fragments_table.php

```diff
@@ -9,11 +10,10 @@ public function up()
     {
         Schema::create('combined_fragments', function (Blueprint $table) {
             $table->increments('id');
-            $table->timestamps();
+            $table->timestamp('timestamps');
```

Comment:
asi som sa ni celkom dobre vyjadril ale čakal som, že si to pozrieš v dokumentácii, ktorú som poslal
$table->timestamps();	// Adds nullable created_at and updated_at columns.
$table->timestamp('added_on');	TIMESTAMP equivalent for the database.